openedx_tagging/core/tagging/models/utils.py

```diff
@@ -4,6 +4,14 @@
 from django.db.models import Aggregate, CharField
 from django.db.models.expressions import Func
 
+# This is NOT the separator of export csv file.
+# This is the separator of the individual group of tags per taxonomy.
+# eg. languages-v1: en;es;fr
+#
+# This character is not allowed to be placed in the value of a tag,
+# to avoid inconsistencies when exporting and importing tags.
+TAGS_CSV_SEPARATOR = ';'
```

Comment:
Actually, would you mind changing this slightly? I want to have a list of invalid characters for tags, like this:

```python
RESERVED_TAG_CHARS = [
    '\t', # Used in the database to separate tag levels in the "lineage" field
          # e.g. lineage="Earth\tNorth America\tMexico\tMexico City"
    '>',  # Used in the search index and Instantsearch frontend to separate tag levels
          # e.g. tags_level3="Earth > North America > Mexico > Mexico City"
    ';',  # Used in CSV exports to separate multiple tags from the same taxonomy
          # e.g. languages-v1: en;es;fr
]
```

Reply:
Updated https://github.com/openedx/openedx-learning/pull/172/commits/3837d6279f66282ba43931497ed381306139a4cb